packages/ember-metal/lib/error_handler.js

```diff
@@ -6,7 +6,7 @@ let getStack = function(error) {
   var stack = error.stack;
   var message = error.message;
 
-  if (stack.indexOf(message) === -1) {
+  if (stack && stack.indexOf(message) === -1) {
```

Comment:
synonymous with the previous behavior where it will Log/dispatch undefined if the error did not have a stack